types/node/test/globals.ts

```diff
@@ -88,3 +88,17 @@ declare var RANDOM_GLOBAL_VARIABLE: true;
     const f: File = {} as any;
     f.name; // $ExpectType string
 }
+
+{
+    crypto.randomUUID(); // $ExpectType `${string}-${string}-${string}-${string}-${string}` || string
```

Comment:
Here you used imported `crypto`, not from global context. Use `globalThis.crypto`

Reply:
Oh, sorry, I confused with `test/crypto.ts`...